### PR TITLE
"logs" page should not load assets from external 3rd party domains

### PR DESCRIPTION
**Pull request by eflyjason**
See https://phabricator.wikimedia.org/T133644

### CODE REVIEW

**Comment by benapetr** [comment]
Hello, did you have a chance to test if it works? Or should we test in production? :)

**Comment by eflyjason** [comment]
I tried to download the site (using Chrome) and replace the modified code and it is at least working fine for me. 😅

File: public_html/logs/index.php [hunk 1]
```diff
@@ -21,17 +21,15 @@
...
 
 $html = new HtmlPage("Wikimedia IRC logs browser");
-googlefonts_init($html);
 $html->Style->items['*']['font-family'] = 'Open Sans';
 
 // Github ribbon
-$github_ribbon = new GitHub_Ribbon();
-$github_ribbon->Repository = "benapetr/wikimedia-bot/tree/master/public_html/logs";
-$html->AppendHtml($github_ribbon->ToHtml());
+$html->AppendHtml('<a class="github-fork-ribbon right-top" href="https://github.com/benapetr/wikimedia-bot/tree/master/public_html/logs" data-ribbon="Fork me on GitHub" title="Fork me on GitHub">Fork me on GitHub</a>');
```
Comment (pppery) [inline public_html/logs/index.php]: Isn't github.com an external site?

Comment (eflyjason) [reply public_html/logs/index.php]: It's in a href, not script or link or img that are "assets" and will not automatically loaded.

**Comment by benapetr** [comment]
let's hope ;)

**Pull request merged**